src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -120,7 +120,7 @@ class SaveInProgressIntro extends React.Component {
           alert = (
             <div className="usa-alert usa-alert-info background-color-only schemaform-sip-alert">
               <div className="schemaform-sip-alert-title">
-                <H className="usa-alert-heading vads-u-font-size--h3">
+                <H className="usa-alert-heading vads-u-font-size--h2">
```

Comment:
Not sure how this will effect other parts of the app.  Can make this prop based if need be.

Reply:
This would most likely affect other applications in ways they don't expect. @bkjohnson or @micahchiang do you have opinions here?

Reply:
This looks like it would just be a visual change. Whatever the semantic level of the `<H>` is wouldn't change.

Reply:
You can set the heading level through a prop:

https://github.com/department-of-veterans-affairs/vets-website/blob/047776d00637ed1a3d343ec0d8f51ad28413af43/src/platform/forms/save-in-progress/SaveInProgressIntro.jsx#L118

Reply:
@bkjohnson Will make the change.  Also, I updated the link to the issue.